src/guides/v2.3/install-gde/install/cli/dev_options.md

```diff
@@ -9,12 +9,14 @@ functional_areas:
 The following topics apply to you *only* if you used `git clone` to install the {{site.data.var.ce}} GitHub repository. This usually means you contribute code to the {{site.data.var.ce}} codebase.
 
  {:.bs-callout-info}
-If you clone the Magento 2 GitHub repository, you <em>cannot</em> use the Magento software in a production environment. You cannot have a live store that accepts orders and so on.
+If you clone the Magento 2 GitHub repository, you <em>cannot</em> use the Magento software in a production environment (a live store that accepts orders).
 
-*  To [update the Magento software]({{ page.baseurl }}/install-gde/install/cli/dev_update-magento.html), use `git pull origin` and `composer update`, then update the Magento database
+*  To get started with your Magento installation or update, ensure your server meets [our system requirements]({{ page.baseurl }}/install-gde/prereq/dev_install.html#prerequisites), [create the Magento file system owner]({{ page.baseurl }}/install-gde/prereq/dev_install.html#prerequisites), [install Composer]({{ page.baseurl }}/install-gde/prereq/dev_install.html#instgde-prereq-compose-install), and [clone the repository]({{ page.baseurl }}/install-gde/prereq/dev_install.html#instgde-prereq-compose-clone).
+*  To ensure your system can run the Magento application successfully, [resolve any dependencies]({{ page.baseurl }}/install-gde/install/prepare-install.html) before you proceed with installation.
+*  To [update the Magento software]({{ page.baseurl }}/install-gde/install/cli/dev_update-magento.html), use `git pull origin` and `composer update`, then update the Magento database.
```

Comment:
Too much detail for a summary topic. Let the linked topic discuss specific commands. Rewrite to something like

To update the Magento software, pull in changes from the Magento 2 GitHub repository, update dependencies, and update the Magento database.

Same comment below.